pkgs/development/tools/ilspycmd/default.nix

```diff
@@ -1,52 +1,60 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, buildDotnetModule
-, dotnetCorePackages
-, powershell
-, autoSignDarwinBinariesHook
-, glibcLocales
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  powershell,
+  autoSignDarwinBinariesHook,
+  glibcLocales,
 }:
-buildDotnetModule rec {
+buildDotnetModule (finalAttrs: {
   pname = "ilspycmd";
   version = "8.0";
 
   src = fetchFromGitHub {
     owner = "icsharpcode";
     repo = "ILSpy";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-ERBYXgpBRXISfqBSBEydEQuD/5T1dvJ+wNg2U5pKip4=";
   };
 
-  nativeBuildInputs = [
-    powershell
-  ] ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64) [ autoSignDarwinBinariesHook ];
+  nativeBuildInputs =
+    [
+      powershell
+    ]
+    ++ lib.optionals (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64) [
+      autoSignDarwinBinariesHook
+    ];
 
   # https://github.com/NixOS/nixpkgs/issues/38991
   # bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
   env.LOCALE_ARCHIVE = lib.optionalString stdenv.hostPlatform.isLinux "${glibcLocales}/lib/locale/locale-archive";
 
-  dotnet-sdk = dotnetCorePackages.sdk_6_0;
-  dotnet-runtime = dotnetCorePackages.runtime_6_0;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
```

Comment:
Can you rename or split https://github.com/NixOS/nixpkgs/commit/314ceb9874ec9d099639cc90a71488765aaa7a57? The commit message doesn't include the important .NET version bump.